metadata.py

```
@@ -27,9 +27,9 @@
 
 def find_credit_pages(path: str) -> Tuple[str, str]:
```

Ruff (0.11.9)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)